charts/csm-installer/README.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# Dell Community Helm Chart for Dell EMC Container Storage Modules (CSM)
+
+Dell EMC Container Storage Modules (CSM) can be deployed using Helm.
+
+For complete deployment instructions, refer to the [Dell EMC Container Storage Modules (CSM) Github repository](https://github.com/dell/csm).
+
+## Helm Chart Versioning
+
+For an explanation and examples on versioning/releasing the Karavi Observability Helm chart, please see the [contributing guide](../../docs/CONTRIBUTING.md#helm-chart-release-strategy).
```

Comment:
References "Karavi Observability".  Is this link still even valid? If so is the content valid?

Reply:
Removed reference to Karavi Observability. Yes, the link and content are still valid detailing how to update versions of a helm chart: https://github.com/dell/helm-charts/blob/main/docs/CONTRIBUTING.md#helm-chart-release-strategy